web-client/public/js/setup-handlers.js

```diff
@@ -19,4 +19,19 @@ export const setupHandlers = grnState => {
         grnState.dashedLine = !$(GREY_EDGES_DASHED_MENU).prop("checked");
         updateApp(grnState);
     });
+
+    $("#normalization-button").click(function () {
+        grnState.normalizationMax = $("#normalization-max").val();
+        updateApp(grnState);
+    });
+
+    $("#reset-normalization-factor-menu, #resetNormalizationButton").click(function () {
+        grnState.normalizationMax = grnState.resetNormalizationMax;
+        updateApp(grnState);
+    });
+
+    $("#edge-weight-normalization-factor-menu").on("change", function () {
+        grnState.normalizationMax = $("#edge-weight-normalization-factor-menu").val();
+        updateApp(grnState);
+    });
```

Comment:
Good, this is the correct C -> M direction.

If a block of handlers looks related and appears to be lengthy, feel free to consolidate that into a single function (e.g., `setupNormalizationFactorHandlers`). It doesn't need to be in a different module the way the loaders/importers got moved. I leave the judgment call of when to do this up to you. Let's see if we agree on when to cross this threshold 🙂